### DISCUSSION TITLE
v0.17.3

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1256] [Releases]
## What's Changed
* [Chore] Refactored Discord notifications to use WebhookCall by @jonezy35 in https://github.com/alexjustesen/speedtest-tracker/pull/1242
* [Bug] Fixed sortable query for json data by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1251
* Refactored Discord Notifications to fix threshold notifications by @jonezy35 in https://github.com/alexjustesen/speedtest-tracker/pull/1254
* Laravel 10.46.0 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1253
* Release v0.17.3 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1255

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.17.2...v0.17.3

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.17.3'>v0.17.3</a>.</em>

**Comment by seh2000** [comment]
Updated now to 0.17.3 no issues...
